src/Bonsai.ML.Torch/ScalarTypeLookup.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch
+{
+    /// <summary>
+    /// Provides methods to look up tensor data types.
+    /// </summary>
+    public static class ScalarTypeLookup
```

Comment:
Should this be `public`? Usually it is unconventional in code to want to parse enum values from strings like this. Is there an issue having it `internal`?